src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute.Fluent/ComputeUsageImpl.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+namespace Microsoft.Azure.Management.Compute.Fluent
+{
+    using Models;
+    using Microsoft.Azure.Management.Resource.Fluent.Core;
+
+    /// <summary>
+    /// The implementation of ComputeUsage.
+    /// </summary>
+    ///GENTHASH:Y29tLm1pY3Jvc29mdC5henVyZS5tYW5hZ2VtZW50LmNvbXB1dGUuaW1wbGVtZW50YXRpb24uQ29tcHV0ZVVzYWdlSW1wbA==
+    internal partial class ComputeUsageImpl :
+        Wrapper<Usage>,
+        IComputeUsage
+    {
+        ///GENMHASH:98D67B93923AC46ECFE338C62748BCCB:AE70F51B2F784C2A02F858F867A2AA34
+        public ComputeUsageUnit Unit()
+        {
+            return new ComputeUsageUnit(Usage.Unit);
+        }
+
+        ///GENMHASH:A5E7AF81C3FDCACFFFE1D6B50B56161F:C0C35E00AF4E17F141675A2C05C7067B
+        internal ComputeUsageImpl(Usage innerObject) : base(innerObject)
+        {
+        }
+
+        ///GENMHASH:9D196E486CC1E35756FD0BEDAB3F3BE4:D46EE1115C895668AEAF34537890C3E6
+        public int Limit()
```

Comment:
For each model interface, there are two partial implementation classes

1. Impl class generated by the converter that explicitly implements the interface and reside in Domain/InterfacesImpl directory

2. Impl class generated by the converter but we need to modify method bodies.

The Impls that belongs to 2nd category contains ONLY methods (including members those can be properties), the corresponding partial impl in category 1 correctly exposes these members as either methods or properties.

So `Limit()` method is a member of category 2 partial class

and `Limit` property in category 1 impl class wrap the above property 

see https://github.com/Azure/azure-sdk-for-net/blob/4d62a3cdc423259d0ca5050c50dd22153d7d9394/src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute.Fluent/Domain/InterfaceImpl/ComputeUsageImpl.cs#L14